intake/source/base.py

```diff
@@ -273,6 +278,18 @@ def _ipython_display_(self):
     def __repr__(self):
         return self.yaml()
 
+    def is_persisted(self):
+        """The base class does not interact with persistence"""
+        return False
+
+    def has_been_ersisted(self):
```

Comment:
```suggestion
    def has_been_persisted(self):
```

Reply:
👍 
Obviously there should be at least one test driver based on the base class which exercises these methods.